components/ws-daemon/nsinsider/main.go

```diff
@@ -476,6 +478,38 @@ func main() {
 					return os.WriteFile("/proc/sys/net/ipv4/ip_forward", []byte("1"), 0644)
 				},
 			},
+			{
+				Name:  "setup-connection-limit",
+				Usage: "set up network connection rate limiting",
+				Flags: []cli.Flag{
+					&cli.IntFlag{
+						Name:     "limit",
+						Required: true,
+					},
+				},
+				Action: func(c *cli.Context) error {
+					nftcon := nftables.Conn{}
+					limitTable := nftcon.AddTable(&nftables.Table{
+						Family: nftables.TableFamilyIPv4,
+						Name:   "gitpod",
```

Comment:
I think it would be better to move ip-fowarding rules to this table. WDYT? Maybe it is good for another PR to take care of that.

Reply:
Yes, I have thought about encapsulating all the network rules we have within the gitpod table, but I felt that for this PR it is out of scope.

Reply:
🆗